solutions/hw1/task1/vectors.py

```diff
@@ -0,0 +1,27 @@
+import math
+
+from typing import List
+
+Vector = List[float]
+
+
+def scalar(first: Vector, second: Vector) -> float:
+    if len(first) != len(second):
+        raise TypeError("Vectors must be the same length")
+    return sum([x * y for x, y in zip(first, second)])
+
+
+def module(vector: Vector) -> float:
+    return math.sqrt(scalar(vector, vector))
+
+
+def angle(first: Vector, second: Vector) -> float:
+    if len(first) != len(second):
+        raise ArithmeticError("Vectors must be the same length")
```

Comment:
Still this strange type of error. I think you should use `TypeError` like everwhere else.